source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,684 @@ Filters Builders
+
+- Items that cost $0 to $25
+- A hotel with ammenities you want
+- A food critic review that mentions "spicy" 
+
+The :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+class provides static factory methods for all the MongoDB query
+operators. Each method returns an instance of the :ref:`Bson <bson>`
+type, which you can pass to any method that expects a query filter. 
+
+.. tip::
+
+   For brevity, you may choose to import the methods of the ``Filters`` class statically:
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Filters.*;
+
+   The examples below assume this static import.
+
+The following examples show you how to use the static factory methods
+provided by the ``Filters`` class to sort your queries. 
+
+The examples use these sample collections that contains the following
+documents: 
+
+``paint_purchaces``
+
+.. code-block:: json
+
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+   {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+   {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+``binary_numbers``
+
+.. code-block:: json
+
+   { _id: 9, a: 54, binaryValue: "00110110" }
+   { _id: 10, a: 20, binaryValue: "00010100" }
+   { _id: 11, a: 68, binaryValue: "1000100" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+``geo_points``
+
+.. code-block:: json
+
+   { _id: 13, type: "Point", coordinates: [ 2,2 ] }
+   { _id: 14, type: "Point", coordinates: [ 5,6 ] }
+   { _id: 15, type: "Point", coordinates: [ 1,3 ] }
+   { _id: 16, type: "Point", coordinates: [ 4,7 ] }
+
+.. _comparison:
+
+Comparison 
+----------
+
+The comparison operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Comparison Operator
+     - Matches
+      
+   * - ``eq()``
+     - values that are equal to a specified value.
+
+   * - ``gt()``
+     - values that are greater than a specified value.
+      
+   * - ``gte()``
+     - values that are greater than or equal to a specified value.
+
+   * - ``lt()``
+     - values that are less than a specified value.
+
+   * - ``lte()``
+     - values that are less than or equal to a specified value.
+
+   * - ``ne()``
+     - values that are not equal to a specified value.
+
+   * - ``in()``
+     - any of the values specified in an array.
+
+   * - ``nin()``
+     - none of the values specified in an array.
+
+   * - ``empty()``
+     - all the documents.
+
+
+The following code snippet shows how to use a comparison filter:
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Filters.*;
+
+   // <MongoCollection setup code here>
+
+   collection.find(<comparison filter>);
```

Comment:
Looking at this now, I'm not sure this code snippet makes much sense since you show it's usage in the example. This only shows how to use it for `find()`, but you can actually pass it to many other different types of CRUD operations.